client/src/components/ManageAccount/ManageAccount.scss

```diff
@@ -127,9 +128,7 @@
 }
 // Buttons and Forms
 .manage-account-submit-button {
-  display: flex;
-  justify-content: space-between;
-  align-items: center;
+  float: right;
```

Comment:
Ah I thought it'd be fine since the button is already contained inside the flexbox of the modal body. I'll revert it back, but I need to figure out why the google accept privacy policy button extends across the entire modal.

Reply:
`width: fit-content` fixes that!